build-wheelhouse/action.yml

```diff
@@ -92,6 +92,21 @@ runs:
       run: |
         python -m pip install ${{ env.install_target }}
 
+    - name: "Verify library is properly installed"
+      shell: bash
+      run: |
+        library_name=${{ inputs.library-name }}
+
+        library_name="${library_name//-/.}"
```

Comment:
I'm not sure if this will always work. 90% of the cases it will... but I'm guessing some will fail

Reply:
I kind of agree.. but I cannot come up with something better 🤷🏻‍♂️